frontend/src/pages/dashboard/Dashboard.tsx

```diff
@@ -97,28 +97,9 @@ const Dashboard = () => {
                 </button>
               )}
             </div>
-            {/* Status filter for ACTIVE tab*/}
-            {searchParams.get(Filters.STATUS) === Status.ACTIVE && (
+            {searchParams.get(Filters.STATUS) !== Status.PENDING && (
               <StatusFilter
-                statusFilter={RecommitmentStatusFilter.filter((obj) =>
-                  ['ALL', 'NEW', 'MEMBER_COMMITTED'].includes(obj.value),
-                )}
-                searchParams={searchParams}
-                setSearchParams={setSearchParams}
-              />
-            )}
-            {/* Status filter for INACTIVE tab */}
-            {searchParams.get(Filters.STATUS) === Status.INACTIVE && (
-              <StatusFilter
-                statusFilter={RecommitmentStatusFilter.filter((obj) =>
-                  [
-                    'ALL',
-                    'MEMBER_NO_RESPONSE',
-                    'MEMBER_DENIED',
-                    'SUPERVISOR_DENIED',
-                    'OTHER',
-                  ].includes(obj.value),
-                )}
+                statusFilter={searchParams.get(Filters.STATUS) && searchParams.get(Filters.STATUS) === Status.ACTIVE ? ActiveRecommitmentStatusFilter : InactiveRecommitmentStatusFilter}
```

Comment:
@leon-ey  the way you had this previously was fine but whenever we can it is preferable to reduce inline logic so I have changed this to use two separately defined constants.